client/src/partials/budget/new/new_budget.html

```diff
@@ -0,0 +1,197 @@
+<header data-header> 
+  {{ "BUDGET.NEW.TITLE" | translate }}
+</header>
+
+<nav>
+  <div class="pull-left">
+      <ol class="breadcrumb">
+        <li><a href="#/"><span class="glyphicon glyphicon-home"></span></a></li>
+        <li class="active">{{ "BUDGET.NEW.TITLE" | translate }}</li>
+      </ol>
+  </div>
+</nav>
+
+<div class="report">
+  <div ng-switch on="session.mode">
+    <div ng-switch-when="configure">
+      <div class="col-sm-offset-3 col-xs-6">
+        <h3 class="center"><b>{{ "BUDGET.NEW.TITLE" | translate }}</b></h3>
+        <p>{{ "BUDGET.NEW.DESC" | translate }}</p>
+
+        <form class="form budget-analysis-form" novalidate>
```

Comment:
There is no class `form`, unless you have defined it elsewhere.  It's a common mistake I make.